conjur/logic/credential_provider/CredentialProviderTypes.py

```diff
@@ -0,0 +1,7 @@
+from enum import Enum
+
+
+class CredentialProviderTypes(Enum):
```

Comment:
Please provide docstring here

Reply:
do we still need override logic?

Reply:
actually we don't need this logic anymore will remove